docs/collections.md

```diff
@@ -1,49 +1,49 @@
 # Data Discovery (Collections)
 
-openEO strives for compatibility with [STAC](https://github.com/radiantearth/stac-spec) and [OGC WFS 3.0](https://github.com/opengeospatial/WFS_FES) as far as possible. Implementing the data discovery endpoints of openEO should also produce mostly valid STAC and WFS 3.0 responses, including an incomplete compatibility with their APIs (see below).
+openEO strives for compatibility with [STAC](https://github.com/radiantearth/stac-spec) and [OGC WFS](https://github.com/opengeospatial/WFS_FES) as far as possible. Implementing the data discovery endpoints of openEO should also produce valid STAC 0.6 and WFS 3.0 responses, including an incomplete compatibility with their APIs (see below).
 
-!!! warning
-    STAC and OGC WFS 3.0, as well as openEO, are still under development and openEO currently only integrates intermediate dev versions due to time constraints.
-	Therefore, it is very likely that further changes and adjustments will be made in the future.
+**WARNING**: STAC and OGC WFS 3, as well as openEO, are still under development.
+Therefore, it is very likely that further changes and adjustments will be made in the future.
 
 ## Extensions
 
-STAC has several extensions (see their [repository](https://github.com/radiantearth/stac-spec)) that can be used to better describe your data. Clients and server are not required to implement all of them, so be aware that some clients may not be able to read all your meta data.
+STAC has several [extensions](https://github.com/radiantearth/stac-spec/tree/v0.6.2/extensions) that can be used to better describe your data. Clients and server are not required to implement all of them, so be aware that some clients may not be able to read all your meta data.
 
-Some commonly used extensions are:
+Some commonly used extensions that are integrated in the openEO API specification are:
 
-- EO extension (mostly integrated within openEO)
-- Scientific extension (integrated within openEO)
-- Dimensions extension (draft)
+- EO extension
```

Comment:
Just a minor remark: is  "EO" vs "SAR" commonly distinguished? I'd rather contrast "Optical" vs "SAR" vs "Thermal" vs ...

Reply:
Not sure about common sense, I'm just porting the STAC specification here. Unfortunately, STAC specifies EO as *Electro-Optical* and not *Earth Observation* in general, which is a bit confusing. I can rename the title for the schema to 'EO (Electro-Optical)' to make this clear.

Reply:
Yes, that might reduce potential confusion, thanks.

Reply:
Did so, thanks for spotting that.